src/dlgPackageExporter.cpp

```diff
@@ -315,19 +313,18 @@ void dlgPackageExporter::slot_updateLocationPlaceholder()
         path = tr("Export to %1").arg(QStringLiteral("%1/%2.mpackage").arg(getActualPath(), packageName));
     }
 
+    checkToEnableExportButton();
```

Comment:
By calling the method here we avoid the need for the original method to be a second slot connected to the `&QLineEdit::textChanged` signal for the `(QLineEdit) lineEdit_packageName` element.